data/net/minecraft/world/level/levelgen/SurfaceSystem.mapping

```diff
@@ -9,3 +9,8 @@ CLASS net/minecraft/world/level/levelgen/SurfaceSystem
 		ARG 1 location
 	METHOD isStone (Lnet/minecraft/world/level/block/state/BlockState;)Z
 		ARG 1 state
+	METHOD topMaterial (Lnet/minecraft/world/level/levelgen/SurfaceRules$RuleSource;Lnet/minecraft/world/level/levelgen/carver/CarvingContext;Ljava/util/function/Function;Lnet/minecraft/world/level/chunk/ChunkAccess;Lnet/minecraft/world/level/levelgen/NoiseChunk;Lnet/minecraft/core/BlockPos;Z)Ljava/util/Optional;
+		ARG 2 context
+		ARG 3 biomeGetter
```

Comment:
```suggestion
		ARG 3 biomeAccessor
```
This ultimately routes through [`WorldCarver#carveBlock`](https://github.com/ParchmentMC/Parchment/blob/5a72c7dcea6ee1a6f810f11be0d676351a2d9691/data/net/minecraft/world/level/levelgen/carver/WorldCarver.mapping#L33) which already names it thusly.

Reply:
Yeah, but you're also creating a new `SurfaceRules.Context` object in which that parameter is assigned to a field named `biomeGetter`. This way copies existing Mojmaps.

Reply:
Oh, I didn't realize that. That mapping pre-dates surface rules then, could you change `WorldCarver#carveBlock` to also be `biomeGetter` then?

Reply:
Done